src/server/scripts/Outland/zone_zangarmarsh.cpp

```diff
@@ -229,6 +229,46 @@ class npc_kayra_longmane : public CreatureScript
     }
 };
 
+enum AsTheCrowFlies
+{
+    SPELL_WHISPER_AURA_1 = 31774,
+    NPC_YSIEL_WINDSINGER = 17841
+};
+
+class spell_crow_whisper_aura : public SpellScriptLoader
+{
+    public:
+        spell_crow_whisper_aura() : SpellScriptLoader("spell_crow_whisper_aura") { }
+
+        class spell_crow_whisper_aura_SpellScript : public SpellScript
+        {
+            PrepareSpellScript(spell_crow_whisper_aura_SpellScript);
+
+            void HandleEffect(SpellEffIndex effIndex)
```

Comment:
```cpp
            void HandleEffect(SpellEffIndex /*effIndex*/)
```
(if the current content is going to be used, that is)

Reply:
Fixed, just in case. Thanks!